src/components/search/facets/custom-facets.vue

```diff
@@ -32,7 +32,7 @@
       </CustomHeaderTogglePanel>
     </template>
 
-    <template #category-paths="{ facet, selectedFilters }">
+    <template #hierarchical-facet="{ facet, selectedFilters }">
```

Comment:
This is not right, the slot uses the id of the facet:

![image](https://user-images.githubusercontent.com/6597815/197704450-0e07d8db-0570-4128-b1ce-f306c52a6944.png)

So you have to rollback these names